### ISSUE TITLE
Support for Java 21

### ISSUE THREAD

**Issue by melix** [beryx/badass-jlink-plugin#238]
I was trying to upgrade my project to Java 21, but I am facing a blocking issue with this plugin, illustrated in this [build scan](https://scans.gradle.com/s/ki6frfpolztau).

The error is:

```
> Task :jsolex:prepareModulesDir FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':jsolex:prepareModulesDir'.
> Unsupported class file major version 65
```

This, as far as I understand, should be fixable by [upgrading to ASM 9.5 here](https://github.com/beryx/badass-jlink-plugin/blob/master/build.gradle#L48).

Thanks!

**Comment by airsquared** [comment]
Fixed in v3.0.0.

**Comment by melix** [comment]
And verified! Thanks a lot for the quick fix!
